scripts/generate_duplicate_sheet.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+
+import csv
+import itertools
+from datetime import datetime
+
+from bugbug import bugzilla
+from bugbug.models.duplicate import DuplicateModel
+
+start = datetime(2019, 5, 15)
+test_bugs = bugzilla.download_bugs_between(start, datetime.now())
+
+m = DuplicateModel()
+m.train()
+
+csvfile = open("duplicate_predictions.csv", "w", newline="\n")
+
+spamwriter = csv.writer(csvfile, delimiter=",")
+
+spamwriter.writerow(
+    ["bug 1 ID", "bug 1 summary", "bug 2 ID", "bug 2 summary", "prediction"]
+)
+
+REPORTERS_TO_IGNORE = {"intermittent-bug-filer@mozilla.bugs", "wptsync@mozilla.bugs"}
+
+# Warning: Classifying all the test bugs takes a while
+for bug_tuple in itertools.combinations(test_bugs, 2):
+    if (
+        bug_tuple[0]["creator"] in REPORTERS_TO_IGNORE
```

Comment:
You can skip these directly in test_bugs, so you avoid making the combinations altogether